docs/api/view.md

```diff
@@ -0,0 +1,100 @@
+# View
+
+> A View.
+
+Process: [Main](../glossary.md#main-process)
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
+
+```javascript
+// TODO example
```

Comment:
Once `setLayout` is documented, it'd be nice to have an example of horizontal/vertical layouts. Maybe a side-by-side view too.

Reply:
Yep, that gets quite a bit more complex though!